modal_test_support/functions.py

```diff
@@ -274,3 +274,21 @@ def build2(self):
     @method()
     def f(self, x):
         return self._k * x
+
+
+@stub.cls(checkpointing_enabled=True)
```

Comment:
Can you update the class to test for multiple checkpointed enter methods? The test should verify that they are executed in order. Wdyt?